cli.js

```diff
@@ -1,46 +1,68 @@
 #!/usr/bin/env node
 
 import path from 'node:path';
-import { existsSync, readFileSync } from 'node:fs';
-import { FiltersEngine, Request } from '@cliqz/adblocker';
+import { existsSync, readFileSync, writeFileSync } from 'node:fs';
+import { FiltersEngine, Request, ENGINE_VERSION } from '@cliqz/adblocker';
 
 import { BASE_PATH } from './scripts/helpers.js';
 
+const debug = process.env.DEBUG === 'true' ? console.log : () => {};
+
 (async () => {
+  const startTime = Date.now();
   const [, , url] = process.argv;
 
   if (!url) {
     console.error('trackerdb takes URL as first argument');
     process.exit(1);
   }
 
+  // Generate input JSON if missing
   const trackerDBPath = path.join(BASE_PATH, 'dist', 'trackerdb.json');
-
   if (!existsSync(trackerDBPath)) {
-    // generate input JSON if missing
     await import('./scripts/export-json/index.js');
   }
 
-  const rawTrackerDB = JSON.parse(readFileSync(trackerDBPath, 'utf-8'));
+  // Generate binary engine if missing
+  const trackerEnginePath = path.join(
+    BASE_PATH,
+    'dist',
+    `trackerdb_${ENGINE_VERSION}.engine`,
+  );
+  if (!existsSync(trackerEnginePath)) {
+    const rawTrackerDB = JSON.parse(readFileSync(trackerDBPath, 'utf-8'));
+    const engine = FiltersEngine.fromTrackerDB(rawTrackerDB);
+    writeFileSync(trackerEnginePath, engine.serialize());
```

Comment:
can we replace this with 
`await import('./scripts/export-engine/index.js');` maybe?

Reply:
Maybe. I am not sure if the behavior of `export-engine/index.js` is equivalent but it's worth checking if it simplifies the logic.